crypto/rand/rand_lib.c

```diff
@@ -46,7 +46,7 @@ void rand_read_tsc(RAND_poll_fn cb, void *arg)
     unsigned char c;
     int i;
 
-    for (i = 0; i < 10; i++) {
+    for (i = 0; i < TSC_READ_COUNT; i++) {
         c = (unsigned char)(OPENSSL_rdtsc() & 0xFF);
```

Comment:
On additional note, it's only appropriate to add OPENSSL_ia32cap_P check. It's not given that that you have TSC, e.g. you don't in SGX enclave... Well, I don't actually know if it's reflected in capability vector read from within SGX, but I'd be surprised if it's not.